pyodide_build/pywasmcross.py

```diff
@@ -8,14 +8,21 @@
 cross-compiling and then pass the command long to emscripten.
 """
 
+import functools
 import json
 import os
+import re
+import shutil
+import subprocess
```

Comment:
I moved all module imports to the top of the file, rather than placing them within functions. It should not amount to anything and is rather to make my linter happy. :)